src/angular.prefix

```diff
@@ -3,4 +3,4 @@
  * (c) 2010-2016 Google, Inc. http://angularjs.org
  * License: MIT
  */
-(function(window, document, undefined) {
+(function(window) {
```

Comment:
I think that having `undefined` as a parameter, which is not assigned, is actually a clever way to ensure that we have the "real" `undefined` inside the closure. So I am for leaving this as a parameter.